### DISCUSSION TITLE
Disable Edited notes on day note

### DISCUSSION THREAD

**Discussion by hwmaat** [TriliumNext/Trilium#4745] [Q&A] [answered]
How would I disable "edited notes" on a "day note"
If I open a day note the edited notes gets expanded immediately. Although it's a nice feature, I want it expanded only when I click it.

Thanks.

**Comment by meichthys** [comment, marked as answer]
Menu > Options > Appearance > Ribbon Widgets > Edited notes....
![image](https://github.com/zadam/trilium/assets/10717998/5c2ad9a2-1b6a-4f41-9527-9742b4ed7176)

**Reply by hwmaat** [reply]
thanks!
